test/functional/autocmd/tabnewentered_spec.lua

```diff
@@ -480,4 +480,40 @@ describe('tabpage/previous', function()
   --   does_not_switch_to_previous_after_entering_operator_pending('<C-W>g<Tab>'))
   it('does not switch to previous via <C-Tab> after entering operator pending',
     does_not_switch_to_previous_after_entering_operator_pending('<C-Tab>'))
+
+  local function does_not_switch_to_previous_when_editing_command_line(characters, completion_visible)
```

Comment:
Let's not continue these big function names, they are cumbersome to read. 

Same info can be communicated clearly without such long names. E.g. `cmdline_win_prevents_tab_switch` ( "cmdline-win" is the standard name for that mode)

Reply:
Fixed.